app/controllers/inventories_controller.rb

```diff
@@ -6,17 +6,50 @@
 
 class InventoriesController < ApplicationController
   before_action :require_can_view_projects!
-  before_action :require_can_edit_projects!, only: [:edit, :update]
-  before_action :set_inventory, only: [:show, :update, :edit]
-  before_action :set_project, only: [:edit, :update]
+  before_action :require_can_edit_projects!, only: [:new, :create, :edit, :update, :destroy]
+  before_action :set_inventory, only: [:show, :update, :edit, :destroy]
+  before_action :set_project, only: [:edit, :update, :destroy]
+  before_action :set_project_from_params, only: [:new, :create]
 
   include ArelHelper
 
+  def new
+    @inventory = @project.inventories.build
+    @fields = processed_form_fields
+    render layout: 'ajax_modal_rails/content'
```

Comment:
Instead of this, `include AjaxModalRails::Controller` near ArelHelper above.

Reply:
✅